hackathon_site/registration/test_forms.py

```diff
@@ -242,6 +243,21 @@ def test_registration_has_closed(self, mock_is_registration_open):
         self.assertFalse(form.is_valid())
         self.assertIn("Registration has closed.", form.non_field_errors())
 
+    def test_invalid_birthday(self):
+        data = self.data.copy()
+        data["birthday"] = (
+            settings.EVENT_START_DATE - timedelta(days=(settings.MINIMUM_AGE * 365) - 1)
```

Comment:
Ah, this works too! I was thinking you would hard code some dates here and use the override settings decorator to change the minimum age setting just for this test, but this is much smarter (and simpler)